layers/core_checks/cc_ray_tracing.cpp

```diff
@@ -2381,7 +2373,434 @@ bool CoreChecks::ValidateBuildPartitionedAccelerationStructureInfoNV(
+                    {"VUID-VkClusterAccelerationStructureCommandsInfoNV-opMode-10471",
+                     [&accelerationStructure_size](const vvl::Buffer &buffer_state) {
+                         return buffer_state.create_info.size < accelerationStructure_size.accelerationStructureSize;
+                     },
+                     [&accelerationStructure_size]() {
+                         return "If input::opMode is VK_CLUSTER_ACCELERATION_STRUCTURE_OP_MODE_EXPLICIT_DESTINATIONS_NV and "
+                                "input::opType is not VK_CLUSTER_ACCELERATION_STRUCTURE_OP_TYPE_MOVE_OBJECTS_NV"
+                                ", the buffers in dstAddressesArray must have a size equal to or larger than the "
+                                "VkAccelerationStructureBuildSizesInfoKHR::accelerationStructureSize value (" +
+                                std::to_string(accelerationStructure_size.accelerationStructureSize) +
+                                ") returned from "
+                                "vkGetClusterAccelerationStructureBuildSizesNV with same input parameters";
+                     },
+                     [](const vvl::Buffer &buffer_state) {
+                         return "buffer size " + std::to_string(buffer_state.create_info.size) +
+                                " is less than required acceleration structure size";
+                     }},
+                }}};
+
+                skip |= dst_addresses_size_validator.ValidateDeviceAddress(
+                    *this, command_infos_loc.dot(Field::dstAddressesArray).dot(Field::deviceAddress), LogObjectList(device),
+                    command_infos.dstAddressesArray.deviceAddress);
+            }
+        }
+    }
+
+    if (command_infos.dstAddressesArray.deviceAddress != 0 && command_infos.dstAddressesArray.stride < 8) {
+        skip |= LogError("VUID-VkClusterAccelerationStructureCommandsInfoNV-dstAddressesArray-10474", device,
+                         command_infos_loc.dot(Field::dstAddressesArray).dot(Field::stride),
+                         "(%" PRIu64 ") must be greater than or equal to 8", command_infos.dstAddressesArray.stride);
+    }
+
+    if (command_infos.dstSizesArray.deviceAddress != 0 && command_infos.dstSizesArray.stride < 4) {
+        skip |= LogError("VUID-VkClusterAccelerationStructureCommandsInfoNV-dstSizesArray-10475", device,
+                         command_infos_loc.dot(Field::dstSizesArray).dot(Field::stride),
+                         "(%" PRIu64 ") must be greater than or equal to 4", command_infos.dstSizesArray.stride);
+    }
+
+    if (command_infos.srcInfosArray.stride < stride_min) {
+        skip |= LogError("VUID-VkClusterAccelerationStructureCommandsInfoNV-srcInfosArray-10476", device,
+                         command_infos_loc.dot(Field::srcInfosArray).dot(Field::stride),
+                         "(%" PRIu64 ") must be greater than size of %s (minimum: %" PRIu32 ")", command_infos.srcInfosArray.stride,
```

Comment:
```suggestion
                         "(%" PRIu64 ") must be greater than size of %s (%" PRIu32 ")", command_infos.srcInfosArray.stride,
```

reading the error message, the "minimum" part is confusing, really just want the `sizeof()` value